### PR TITLE
repeat and bounce

### PR DESCRIPTION
**Pull request by callumok2004**
old prs show it working
idk why there are 500 other commits :D ignore them

### CODE REVIEW

File: websites/0-9/8muses/dist/metadata.json [hunk 1]
```diff
@@ -23,4 +23,5 @@
     "porn"
   ],
   "category": "other"
-}
\ No newline at end of file
+}
+
```
Comment (StrikerFRFX) [inline websites/0-9/8muses/dist/metadata.json:27]: No newline at end of file, undo.

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]

File: websites/0-9/8muses/dist/metadata.json (hunk 1 above)
Comment (callumok2004) [reply websites/0-9/8muses/dist/metadata.json:27]: lmfao what

**Review by StrikerFRFX — APPROVED** [review]

File: websites/R/Repeat/dist/metadata.json [hunk 2]
```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/I5G1Lvn.png",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:16]: 512x512

File: websites/B/Bounce/dist/metadata.json [hunk 3]
```diff
@@ -11,12 +11,14 @@
...
+    "itsbounce.uk",
+    "wearebounce.co.uk"
   ],
-  "version": "2.5.3",
-  "logo": "https://boun.cc/cdn/noshadow.png",
-  "thumbnail": "https://i.imgur.com/XmJTiZY.png",
+  "version": "3.0.0",
+  "logo": "https://i.imgur.com/JgK4qYg.png",
+  "thumbnail": "https://i.imgur.com/tlLbekj.png",
   "color": "#32a852",
```
Comment (Bas950) [inline websites/B/Bounce/dist/metadata.json:22]: thats not really a promotion image or a screenshot of the website homepage... the colors are not even related to the logo...

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version on the ones with description changes

**Pull request closed without merging**